ansible-ipi-install/roles/node-prep/tasks/70_enabled_fw_services.yml

```diff
@@ -1,19 +1,47 @@
 ---
-- name: Enable Firewall for HTTP
-  firewalld:
-    service: http
-    permanent: yes
-    state: enabled
-  become: yes
+- name: Configure firewalld
+  block:
+    - name: Enable Firewall for HTTP
+      firewalld:
+        service: http
+        permanent: yes
+        state: enabled
+      become: yes
 
-- name: Open Firewall port 5000/tcp, zone libvirt and public
-  firewalld:
-    port: 5000/tcp
-    permanent: yes
-    state: enabled
-    zone: "{{ item }}"
-  become: yes
-  with_items:
-  - libvirt
-  - public
+    - name: Open Firewall port 5000/tcp, zone libvirt and public
+      firewalld:
+        port: 5000/tcp
+        permanent: yes
+        state: enabled
+        zone: "{{ item }}"
+      become: yes
+      with_items:
+      - libvirt
+      - public
+  when: firewall != "iptables"
 
+- name: Configure iptables
+  block:
+    - name: Enable Firewall for HTTP
+      iptables:
```

Comment:
AFAIK the iptables ansible module just modify the 'in memory' iptables rules, you need to store them to make them survive reboots.

Reply:
Good catch. I'll have a look at this.